uaclient/config.py

```diff
@@ -755,6 +791,19 @@ def help(self, name):
         return response_dict
 
     def process_config(self):
+        for prop in (
+            "update_messaging_timer",
+            "update_status_timer",
+            "gcp_auto_attach_timer",
+        ):
+            value = getattr(self, prop)
+            if not isinstance(value, int) or value < 0:
+                error_msg = (
+                    "Value for the {} interval must be an integer. "
+                    "Default value will be used."
+                ).format(prop)
+                logging.error(error_msg)
```

Comment:
I think we can abort the method if we detect this, similar to what we do with the proxy config

Reply:
I remember @mitechie said to avoid failing when dealing with the configuration files. CLI fails and does not set the value, but processing the config should not fail.

Reply:
But this is the user manually running `ua refresh config`. In that situation, I think we should fail early to alert the user about the mistake.